packages/frontend/core/src/blocksuite/ai/_common/chat-actions-handle.ts

```diff
@@ -273,17 +295,57 @@ const INSERT_BELOW = {
     const currentTextSelection = currentSelections.text;
     const currentBlockSelections = currentSelections.blocks;
     const currentImageSelections = currentSelections.images;
-    const [_, data] = host.command.exec(getSelectedBlocksCommand, {
-      currentTextSelection,
-      currentBlockSelections,
-      currentImageSelections,
-    });
-    if (!data.selectedBlocks) return false;
+
+    if (!host.doc.root) {
+      return false;
+    }
+
+    const [_, { isNothingSelected }] = host.command.exec(
```

Comment:
Use `const { selectedBlocks } = getSelections(host)` and detect the `selectedBlocks` length?

Reply:
Considering the future design of BlockSuite, using commands is highly recommended